tls/policy/s2n_policy_feature.h

```diff
@@ -153,3 +153,73 @@ struct s2n_security_policy *s2n_security_policy_build(struct s2n_security_policy
  * @returns S2N_SUCCESS on success. S2N_FAILURE on failure
  */
 int s2n_security_policy_free(struct s2n_security_policy **policy);
+
+/**
+ * Output format types for verbose policy output.
+ */
+typedef enum {
+    /**
+     * FORMAT_V1: Human-readable structured format
```

Comment:
You didn't update the name here. Probably best just not to include the name here.